include/array/array.h

```diff
@@ -2264,12 +2264,39 @@ class array {
    * `alloc`. This is a deep copy of the contents of `other`. */
   array(const array& other, const Alloc& alloc) : array(alloc) { assign(other); }
 
+  /** Move construct from another array `other`. If the allocator of this array
+   * is propagate on container move assignment, this operation moves the allocation
```

Comment:
I think this should use the actual symbol name (`propagate_on_container_move_assignment`), to make it easier to see that it is something that can be googled.

Reply:
Sure, fixed.